src/web/components/SlotBodyEnd.tsx

```diff
@@ -6,24 +6,54 @@ const wrapperMargins = css`
     margin: 18px 0;
 `;
 
-export const SlotBodyEnd = () => {
-    const endpointUrl = 'https://contributions.guardianapis.com/epic';
+type Props = {
+    contentType: string;
+    sectionName?: string;
+    shouldHideReaderRevenue: boolean;
+    isMinuteArticle: boolean;
+    isPaidContent: boolean;
+    tags: TagType[];
+};
 
-    const trackingParams = {
-        ophanPageId: window.guardian.config.ophan?.pageViewId,
-        ophanComponentId: 'ACQUISITIONS_EPIC',
-        platformId: 'GUARDIAN_WEB',
-        campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
-        abTestName: 'remote_epic_test',
-        abTestVariant: 'api',
-        referrerUrl: window.location.href.split('?')[0],
+export const SlotBodyEnd = ({
+    contentType,
+    sectionName,
+    shouldHideReaderRevenue,
+    isMinuteArticle,
+    isPaidContent,
+    tags,
+}: Props) => {
+    // Putting together the request payload
+    const contributionsPayload = {
+        tracking: {
+            ophanPageId: window.guardian.config.ophan.pageViewId,
+            ophanComponentId: 'ACQUISITIONS_EPIC',
+            platformId: 'GUARDIAN_WEB',
+            campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
+            abTestName: 'remote_epic_test',
+            abTestVariant: 'api',
+            referrerUrl: window.location.href.split('?')[0],
+        },
+        localisation: {
+            countryCode: 'GB', // TODO: make this dynamic
+        },
+        targeting: {
+            contentType,
+            sectionName,
+            shouldHideReaderRevenue,
+            isMinuteArticle,
+            isPaidContent,
+            tags,
+        },
     };
+
+    const endpointUrl = 'https://contributions.guardianapis.com/epic';
     const { data, error } = useApi(endpointUrl, {
         method: 'POST',
         headers: {
             'Content-Type': 'application/json',
         },
-        body: JSON.stringify(trackingParams),
+        body: JSON.stringify(contributionsPayload),
```

Comment:
This isn't one for this PR but just so I don't forget: do we need to consider escaping this payload?

Reply:
I believe `JSON.stringify()` will do everything we need.